src/plugins/plugin-leaflet/leaflet-layer-leaflet.js

```diff
@@ -83,6 +83,19 @@ const LeafletLayer = (layerModel) => {
       }
       layer = new ClusterLayer(layerModel);
       break;
+    case 'vector':
+      if (
+        JSON.stringify(layerConfigParsed.body).indexOf('style: "function') >= 0
+      ) {
+        layerConfigParsed.body.style = eval2(
```

Comment:
Why do we need `eval2`?

Reply:
First: you want to be a maintainer, doesn't it? 😈 
Second: it is a very old code, that our way to convert a string to a javascript object. And `JSON.parse` doesn't valid for this.

Reply:
Haha no, I'm happy with the current arrangement. It's @edbrett fault for adding me in the PR, I get notifications. And @tiagojsag fault for teaching me how to be nosy 😛 .

I just ask because eval2 can be a vulnerability, I was taught to question the use of `eval`. Thanks for explaining! I will leave now 🚶

Reply:
It's ok, feel free to be noisy :). Btw you are right, do you know any alternative?